### PR TITLE
[-] Improvement - Reduce the number of http requests to init the project

### PR DESCRIPTION
**Pull request by VincentMolinie**
Linked to: https://github.com/ForestAdmin/forestadmin-server/pull/709

### CODE REVIEW

**Comment by VincentMolinie** [comment]
[(5) Onboarding - Users want to read the onboarding steps for any stack](https://trello.com/c/J1FxP9Bf/3142-5-onboarding-users-want-to-read-the-onboarding-steps-for-any-stack)

**Comment by ArnaudValensi** [comment]
We need to wait until we decide what to do.

**Comment by VincentMolinie** [comment]
We choose to throw an error if the project as the same name. So lumber catch the error retrieve the project if possible

**Comment by VincentMolinie** [comment]
[Onboarding - The onboarding of a project is done after signup (Feature Mastercard)](https://trello.com/c/ZHgTZz4d/3155-onboarding-the-onboarding-of-a-project-is-done-after-signup-feature-mastercard)

File: lumber-generate.js [hunk 1]
```diff
@@ -18,14 +18,23 @@ function isDirectoryExist(path) {
...
   .option('-c, --connection-url', 'Enter the database credentials with a connection URL')
   .option('--no-db', 'Use Lumber without a database.')
+  .action((projectNameArg) => { projectName = projectNameArg; })
   .parse(process.argv);
 
 (async () => {
-  const config = await Prompter(program, [
+  if (!authenticator.getAuthToken()) {
+    logger.error('💀  Oops, you need to be logged in to execute this command. 💀 Try the "lumber login" command.');
+    process.exit(1);
```
Comment (arnaudbesnier) [inline lumber-generate.js]: As a new user, I am stuck here and cannot do anything with the CLI, unless I create a user using ForestAdmin onboarding.

**Pull request merged**